turbo.json

```diff
@@ -25,6 +25,10 @@
     "build": {
       "dependsOn": ["^build"],
       "outputs": ["dist/**"]
+    },
+    "build:docs": {
```

Comment:
for website project, you have to specifiy it depends on build:docs of dependencies

that's why test are ko